### PR TITLE
Fix MD formatting issues on Multiple documents

### PR DESCRIPTION
**Pull request by bmfmancini**

### CODE REVIEW

File: General-Installing-Instructions.md [hunk 1]
```diff
@@ -89,6 +89,14 @@ The following modules are optional, but preferred to be installed.
 
 - com or dotnet (windows only)
 
+### A special note for systems using PHP-FPM
+ Prior to starting the setup process of Cacti you should restart the PHP-FPM
+ Daemon to rebuild the Cache or you may recived a HTTP 500 Error
+   
+  ```console
+  systemctl restart php-fpm
+  ```
```
Comment (netniV) [inline General-Installing-Instructions.md]: incorrect indentation

File: Installing-Under-Ubuntu-Debian.md [hunk 2]
```diff
@@ -7,6 +7,14 @@ apt-get update
 apt-get install -y apache2 rrdtool mariadb-server snmp snmpd php7.0 php-mysql php7.0-snmp php7.0-xml php7.0-mbstring php7.0-json php7.0-gd php7.0-gmp php7.0-zip php7.0-ldap php7.0-mc
 ```
 
+### A special note for systems using PHP-FPM
+ Prior to starting the setup process of Cacti you should restart the PHP-FPM
+ Daemon to rebuild the Cache or you may recived a HTTP 500 Error
+   
+   ```console
+   systemctl restart php-fpm
+   ```
```
Comment (netniV) [inline Installing-Under-Ubuntu-Debian.md]: incorrect indentation

**Review by netniV — CHANGES_REQUESTED** [review]

**Comment by bmfmancini** [comment]
@netniV  I have updated the spacing

File: General-Installing-Instructions.md [hunk 3]
```diff
@@ -89,6 +89,14 @@ The following modules are optional, but preferred to be installed.
 
 - com or dotnet (windows only)
 
+### A special note for systems using PHP-FPM
+Prior to starting the setup process of Cacti you should restart the PHP-FPM
+Daemon to rebuild the Cache or you may recive a HTTP 500 Error
+   
+```console
+ systemctl restart php-fpm
```
Comment (netniV) [inline General-Installing-Instructions.md]: space :)

File: Install-Under-CentOS_LAMP.md [hunk 4]
```diff
@@ -19,6 +19,13 @@
    dnf module enable php:8.0
    ```
 
+### A special note for systems using PHP-FPM
+Prior to starting the setup process of Cacti you should restart the PHP-FPM
+Daemon to rebuild the Cache or you may recive a HTTP 500 Error
+   
+```console
+ systemctl restart php-fpm
```
Comment (netniV) [inline Install-Under-CentOS_LAMP.md]: space :)

File: Installing-Under-Ubuntu-Debian.md (hunk 2 above)
Comment (netniV) [reply Installing-Under-Ubuntu-Debian.md]: Still incorrect.

**Review by netniV — CHANGES_REQUESTED** [review]

**Comment by TheWitness** [comment]
Is this ready to merge?

**Review by TheWitness — DISMISSED** [review]

**Comment by netniV** [comment]
No, still outstanding indentation issues :)

File: Cacti-SSL-Configuration.md [hunk 5]
```diff
@@ -1,41 +1,36 @@
...
 openssl x509 -req -days 700 -in ca.csr -signkey ca.key -out ca.crt
 cp ca.crt /etc/pki/tls/certs
 cp ca.key /etc/pki/tls/private/ca.key
 cp ca.csr /etc/pki/tls/private/ca.csr
-```
 
-Then we need to update the Apache SSL configuration file
+Then we need to update the Apache SSL configuration file:
 
-```console
```
Comment (netniV) [inline Cacti-SSL-Configuration.md:20]: You seem to have lost the example code markers

File: Cacti-SSL-Configuration.md [hunk 6]
```diff
@@ -1,41 +1,36 @@
...
 
-![cacti-ssl](images/cacti-https.PNG)
+cacti-ssl
 
+### Note: If your Cacti system is public
 
-#### Note: If your Cacti system is public it is recommended to get a certificate from a trusted certificate provider
+it is recommended to get a certificate from a trusted certificate provider.
 
----
```
Comment (netniV) [inline Cacti-SSL-Configuration.md:40]: Missing separator

File: Cacti-SSL-Configuration.md [hunk 7]
```diff
@@ -1,41 +1,36 @@
...
+After configuring the web server to accept https, you can enable https in the GUI
+
+## Note, if using multiple pollers
+
+all must have HTTPS enabled for the remote polling feature to work properly.
 
-![cacti-ssl](images/cacti-https.PNG)
+cacti-ssl
 
+### Note: If your Cacti system is public
```
Comment (netniV) [inline Cacti-SSL-Configuration.md]: This is not a full title and looks wrong.

File: Cacti-SSL-Configuration.md [hunk 8]
```diff
@@ -1,41 +1,36 @@
...
 
-```console
 systemctl restart httpd
-```
 
-After you have configured the web server to accept https you can force https in the Cacti GUI
-#### Warning if you are using multiple pollers ensure all pollers have HTTPS enabled or remote polling feature will not work
+After configuring the web server to accept https, you can enable https in the GUI
+
+## Note, if using multiple pollers
```
Comment (netniV) [inline Cacti-SSL-Configuration.md]: This is not a title and looks wrong

**Review by netniV — CHANGES_REQUESTED** [review]

File: Cacti-SSL-Configuration.md [hunk 9]
```diff
@@ -3,6 +3,8 @@
 Enabling SSL for Cacti is mostly done at the webserver level.
 An example SSL config for Http is as follows:
 
+```bash
+
```
Comment (netniV) [inline Cacti-SSL-Configuration.md:7]: Extra space at start of bash code

File: Cacti-SSL-Configuration.md [hunk 10]
```diff
@@ -11,19 +13,23 @@ cp ca.crt /etc/pki/tls/certs
...
 Restart the httpd service:
 
 systemctl restart httpd
 
 After configuring the web server to accept https, you can enable https in the GUI
 
-## Note, if using multiple pollers
+### Note, if using multiple pollers
 
 all must have HTTPS enabled for the remote polling feature to work properly.
```
Comment (netniV) [inline Cacti-SSL-Configuration.md:34]: I think we should update this to something like 

# Remote Poller Setup

When using remote pollers, you MUST enable SSL between the main and remote pollers.  Cacti will not function without secure communications.

**Review by netniV — CHANGES_REQUESTED** [review]
Couple of suggestions.  This review was based on only the latest commit.  It didn't take into account issues that still persisted.

File: Cacti-SSL-Configuration.md [hunk 11]
```diff
@@ -1,41 +1,42 @@
...
 SSLCertificateFile /etc/pki/tls/certs/ca.crt
 SSLCertificateKeyFile /etc/pki/tls/private/ca.key
 ```
 
-restart the httpd service
+Restart the httpd service:
 
-```console
 systemctl restart httpd
-```
```
Comment (netniV) [inline Cacti-SSL-Configuration.md:30]: Missing code markers

File: Data-Source-Templates.md [hunk 12]
```diff
@@ -26,72 +26,96 @@ You will also need to tell the template what the data source type is.
...
+    In database software, such data sets are referred to as "virtual" or "computed"
+    columns.
 
 ![Data-Source-Templates](images/datasource-template-create.png)
 
 You are also able to use Variables to set the name for the data source.
 The available Variables can be seen here
 
-https://github.com/Cacti/documentation/blob/develop/Variables.md
+<https://github.com/Cacti/documentation/blob/develop/Variables.md>
```
Comment (netniV) [inline Data-Source-Templates.md:118]: This seems to be a link but is done as an HTML tag?  Is this correct?

File: General-Installing-Instructions.md [hunk 13]
```diff
@@ -89,6 +89,15 @@ The following modules are optional, but preferred to be installed.
 
 - com or dotnet (windows only)
 
+### A special note for systems using PHP-FPM
+
+Prior to starting the setup process of Cacti you should restart the PHP-FPM
+Daemon to rebuild the Cache or you may recive a HTTP 500 Error
+
+ ```console
+   systemctl restart php-fpm
```
Comment (netniV) [inline General-Installing-Instructions.md:98]: Space indentations

File: Graph-Templates.md [hunk 14]
```diff
@@ -29,19 +29,31 @@ Option | Description
...
+*Graph Item Inputs* | This list is created automatically when you add **Data Sources**
+ to a **Graph Template**.  However, you can also override certain **Data Template**
+ and **Graph Template** fields by adding specifically named objects to this
+section as well.
+*Graph Template* | Allows you to name the **Graph Template**, and whether or not
+you will allow multiple instances of this **Graph Template** to be used for a
+**Device**, and whether or not to test for valid data before allowing the creation
+of **Graphs**.  The *Test Data Source* option is useful when you want to leverage
+complex **Device Template** that contains **Graph Templates** that may not be applicable
+for all classes of **Devices** using the **Graph Template**.
```
Comment (netniV) [inline Graph-Templates.md:52]: This is one kinds of area where sticking to the 80 column limit should be avoided.  Tables are not so easy to read when they are split like this.

File: Graph-Templates.md [hunk 15]
```diff
@@ -79,25 +109,26 @@ mind when creating a **Graph Template**.
 
 ## Graph Template - General and Common Options
 
-- **Name** - This is the name that you provide to the **Graph Template**.  Since Cacti
+- **Name** - This is the name that you provide to the **Graph Template**.Since Cacti
```
Comment (netniV) [inline Graph-Templates.md:112]: Missing space?

File: Graph-Templates.md [hunk 16]
```diff
@@ -79,25 +109,26 @@ mind when creating a **Graph Template**.
...
-can really slow the **Graph** creation process.  However, it's quite useful if
-you have a hybrid **Device Template** that includes a myriad of **Graph Templates**
-that may not work with all **Devices** that you create using the **Device Template**.
+- **Test Data Sources** - This option will test that the **Data Source** returns
+good data before allowing the creation of the **Graph**.
+You should be aware that if your **Data Source** calls a script that runs
+for a long time, selecting this option can really slow the **Graph** creation process.
+ However, it's quite useful if you have a hybrid **Device Template** that includes
+ a myriad of **Graph Templates** that may not work with all **Devices**
+ that you create using the **Device Template**.
```
Comment (netniV) [inline Graph-Templates.md:126]: Indentation is wrong

File: Install-Under-CentOS_LAMP.md [hunk 17]
```diff
@@ -19,14 +19,22 @@
...
+### A special note for systems using PHP-FPM
 
-2. Install Apache
+Prior to starting the setup process of Cacti you should restart the PHP-FPM
+Daemon to rebuild the Cache or you may recive a HTTP 500 Error
+
+   ```console
+   systemctl restart php-fpm
+   ```
+
```
Comment (netniV) [inline Install-Under-CentOS_LAMP.md:30]: Incorrect indentation

File: Install-Under-CentOS_LAMP.md [hunk 18]
```diff
@@ -19,14 +19,22 @@
...
+2.Install Apache
 
    ```console
    yum install -y httpd
    ```
 
-3. Enable and start the service to ensure it starts when the system reboots
+3.Enable and start the service to ensure it starts when the system reboots
```
Comment (netniV) [inline Install-Under-CentOS_LAMP.md:37]: Space missing

File: Install-Under-CentOS_LAMP.md [hunk 19]
```diff
@@ -228,7 +236,8 @@ during the installation.
    and `your_cacti_password` with your own details.
 
    ```sql
-   MariaDB [(none)]> CREATE USER 'your_cacti_username'@'localhost' IDENTIFIED BY 'your_cacti_password';
+   MariaDB [(none)]> CREATE USER 'your_cacti_username'@'localhost' 
+   IDENTIFIED BY 'your_cacti_password';
```
Comment (netniV) [inline Install-Under-CentOS_LAMP.md:240]: Did not need splitting

**Review by netniV — CHANGES_REQUESTED** [review]
This PR is messy with the bad changes from the editor that didn't care properly about rulesets.  It may be worth creating a diff, checking for actual changes vs format changes, then reapply the actual changes to a fresh branch.  After that, applying any formatting rules as necessary so that things like indentation and breaks are properly applied.  

Reach out to me if you have any questions.

**Comment by TheWitness** [comment]
@bmfmancini, can you work on the outstanding comments.  @netniV, if they are resolved, can you mark such so that we can merge?

**Comment by bmfmancini** [comment]
Netniv was going to look at this one from what I remember differences
between the MD standards and our doc standards

**Comment by netniV** [comment]
Yeah I was trying to find a way to make it easier for @bmfmancini to comply with our normal doc standards using an editor such as VSC.  i'll try and get back on that this week.

**Comment by TheWitness** [comment]
Where is this at?

**Comment by netniV** [comment]
Checkout Pull 177, which has seans changes, plus my improvements and should close this one when merged.

**Pull request closed without merging**